Jenkinsfile

```diff
@@ -1,6 +1,8 @@
 pipeline {
     agent {
-        dockerfile true
+        dockerfile {
+            args '-v $HOME/.gradle:/root/.gradle'
```

Comment:
This lets us mount the Gradle cache from the Jenkins host node, dramatically speeding up builds